addons/oceanbase-cluster/templates/cluster.yaml

```diff
@@ -11,6 +11,18 @@ spec:
   clusterVersionRef: "oceanbase-{{ default .Chart.AppVersion .Values.clusterVersionOverride }}"
   terminationPolicy: {{ .Values.extra.terminationPolicy }}
   # {{- include "kblib.affinity" . | indent 2 }}
+  {{- if gt (int .Values.obClusters) 1 }}
+  services:
+    - name: rw-svc
+      spec:
+        ports:
+          - name: sql
+            port: 2881
+            targetPort: sql
+            protocol: TCP
+        selector:
+          kubeblocks.io/role: primary
```

Comment:
It would be better to use the outer RoleSelector. kubeblocks.io/role: is a key for internal implementation.